src/Evolution/DiscontinuousGalerkin/Limiters/MinmodHelpers.cpp

```diff
@@ -73,9 +73,8 @@ double effective_difference_to_neighbor(
     const Side& side,
     const DirectionMap<VolumeDim, double>& effective_neighbor_means,
     const DirectionMap<VolumeDim, double>& effective_neighbor_sizes) noexcept {
-  const auto& externals = element.external_boundaries();
   const auto dir = Direction<VolumeDim>(dim, side);
-  const bool has_neighbors = (externals.find(dir) == externals.end());
+  const bool has_neighbors = element.neighbors().contains(dir);
```

Comment:
How does this change help with boundary conditions, are you planning to make changes to `element.external_boundaries()` or `element.neighbors()`?

Reply:
@nilsdeppe is thinking of changing/removing `element.external_boundaries()`. The code change here means one less place to worry about side effects... and is also cleaner and easier to understand, so a good improvement regardless.

Reply:
just getting the external boundaries without their boundary conditions isn't a useful operation. Or, at least, we don't do it anywhere in the code and I don't see why you would need it. This means I want to remove `element.external_boundaries` and replace `block.external_boundaries()` with `block.boundary_conditions()`.